### PR TITLE
Update rubygem-foreman_scc_manager to 2.0.0

### PR DESCRIPTION
**Pull request by m-bucher**
Introducing new SCC-product selection page.

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Thanks! Minor comment inline.

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 1]
```diff
@@ -8,8 +8,8 @@
 %global katello_min_version 3.7.0
```
[the remark this answers is not in the document]
Comment (m-bucher) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:8]: Makes sense :sweat_smile:

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 2]
```diff
@@ -80,7 +86,7 @@ cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 %foreman_bundlerd_file
-%foreman_precompile_plugin -s
+%foreman_precompile_plugin -a -s
```
Comment (m-bucher) [inline packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:89]: Looks like `-a` is only used by this plugin and foreman_snapshot_management. Is this still relevant?

Comment (ekohl) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:89]: The comment for what it does:
https://github.com/theforeman/foreman-packaging/blob/37b3a9642d73d24026d05f9d95ee45f5144185ba/packages/foreman/foreman/foreman.spec#L850-L854
Having said that, I thought we dropped the apipie cache and always generated it on the fly now. 8509b32e57ce91b949d8a83f40f9ff69e3748186 did make that change for Foreman itself, but not for the plugins.

@ehelms @evgeni am I missing something, or is this an oversight?

Comment (evgeni) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:89]: Looks like an oversight. I think we wanted to keep `-a` as an noop (did we?) but the compilation itself can really be dropped. The bits are unused anyways.

Comment (m-bucher) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: OK, dropped it :grin:

Comment (evgeni) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: Or maybe we just dropped all `-a` calls and called it a day?

**Review by ekohl — APPROVED** [review]

**Pull request merged**